docs/tests.md

```diff
@@ -56,7 +56,7 @@ Run the following (from the root of the repository):
 
 ```
 make envtest
-./bin/setup-envtest 1.26
+./bin/setup-envtest use 1.26
```

Comment:
Can you please explain what does this change do?

Reply:
it just corrects wrong usage of application